tests/integration/records/table_validator.py

```diff
@@ -6,7 +6,11 @@
 from sqlalchemy.sql.elements import TextClause
 from typing import Optional, Dict, Any, Union, List
 from .timezone import set_session_tz
-from .expected_column_types import expected_column_types
```

Comment:
This file, one of the 3 largest in the project, increased in size to 439 lines.  The total size of those files is now 3481 lines (target: 897).  Is this file complex enough to refactor?